doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -571,6 +571,10 @@ Other libraries
 
     * Added the :file:`ncs/nrf/subsys/partition_manager/pm.yml.pgps` file.
 
+  * :ref:`nrf_profiler`:
+
+      * Renamed profiler to nrf_profiler.
```

Comment:
Technically speaking, the RST file is still titled "Profiler", so this change only took place in the code.

Reply:
Changed to nRF Profiler in documentation